src/IndexUpdater.php

```diff
@@ -106,14 +135,14 @@ public function updateProperties(array $properties): void
     public function deleteProperty(array $property): void
     {
         $name = $property['name'];
-        $type = $property['type'];
+        $typeOrId = $property['type'];
         $parentClasses = $property['parentClasses'];
 
-        if (empty($name) || empty($type)) {
+        if (empty($name) || empty($typeOrId)) {
```

Comment:
Same confusion as before, we are checking value with a variable that can be both Type Or Id. I got the idea but visually it is really weird